tor_ocr/main.py

```diff
@@ -245,11 +258,14 @@ def run(config):
         _(base_comment.format(result['process_time_in_ms'] / 1000))
     )
 
-    for chunk in chunks(result['text'], 9000):
+    for counter, chunk in enumerate(chunks(result['text'], 9000)):
+        # grab first comment for transcription url
+        if counter==0:
```

Comment:
Will putting it here grab the introduction comment or the first comment of the generated transcription?

Reply:
good catch, its getting the intro comment there. I moved it back and tested to make sure its getting the first transcription comment.